README.md

```diff
@@ -288,13 +287,13 @@ If a `Channel message` is received that reads `ATTENTION: All staff report to Sa
 
 If HexChat is started more than once, and this plugin is loaded in each instance, settings set in any HexChat window are saved, but those changes aren't picked up by the other clients.
 
-Reload the plugin to update thaat window:
+Reload the plugin to update that window:
 
 ```
 /lua reload specialsounds.lua
 ```
 
-Using a different file name if it's not installed to HexChat's `config/addons/` folder.
+Use a different file name if it's not installed to HexChat's `config/addons/` folder.

```

Comment:
Elaborate?

Reply:
I meant to address the scenario where HexChat cannot locate the plugin.

Reply:
Yes, but "Use a different filename" doesn't seem to be that helpful... as in it gives almost 0 direction what to actually do, other than "do something else"

Reply:
How does the following sound, inserted in the [Installation](https://github.com/2ndBillingCycle/specialsounds/blob/7196708d51ca2c52979c4dd800aedebac2087ac3/README.md#installation) section?

---

Sometimes, HexChat does not find the file:

```
Lua syntax error: cannot open C:\hexchat\config\addons\specialsound.lua: No such file or directory
```

If this happens, or if `SpecialSounds.lua` is not downloaded to the `config/addons/` directory, give the full path to the file. If this has spaces, wrap the path in `"`:

```
/lua load C:\Users\2nd\Downloads\SpecialSounds.lua
```

Reply:
Perfect.